_people/tunacan.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Tunacan Jones
+image: tunacan-jones.jpg
+---
+
+What can you say about Tunacan?
```

Comment:
I've created a file for the "person". You'll see it in the `_people` directory. You'll notice the file I've added down below and it resides in the `images/people/` directory. The code knows to look there for people photos and so we only have to give the filename.